### DISCUSSION TITLE
Link to document in sidebar B from sidebar A

### DISCUSSION THREAD

**Discussion by compojoom** [facebook/docusaurus#3929] [General] [answered]
I've configured docusaurus to support docs for multiple components by using the plugins options like this:
```
  plugins: [
    [
      "@docusaurus/plugin-content-docs",
      {
        id: "blockchain",
        path: "blockchain",
        routeBasePath: "blockchain",
        sidebarPath: require.resolve("./sidebars_blockchain.js"),
      },
    ],
    [
      "@docusaurus/plugin-content-docs",
      {
        id: "mobileapp",
        path: "mobileapp",
        routeBasePath: "mobileapp",
        sidebarPath: require.resolve("./sidebars_mobileapp.js"),
      },
    ],
```

now in the sidebars_mobileapp I would like to link to a document in the blockchain docs. But docusaurus says document ids don't match and they don't match because it looks only in the mobileapp folder :)

Currently I hacked this by using a href type instead of a doc link, but it would be nice if one could achieve this with a doc type entry.

**Comment by slorber** [comment, marked as answer]
Hi,

This is unlikely to be supported in the future

2 docs plugin are 100% independant, they don't access the data from the other one.

Also 2 docs plugin instances might have different versioning schemes (ios v1/v2 vs android v3/v4 for example)
If ios v2 sidebar did link to an android doc id, we would not be able to know to which android doc is version to link to (v3 vs v4).
Using a link instead of a doc id makes it explicit which version you want to target exactly.

If you don't have different versioning schemes for your mobile app and blockchain, you should rather just use a single doc plugin instance instead (you can still use 2 distinct sidebars in same plugin instance).

**Comment by compojoom** [comment]
Ok, I see. 
No everything is versioned, that's why they are separate components.

Thanks!
